### PR TITLE
cilium status: fix --brief to print less when cilium isn't running

### PR DESCRIPTION
**Pull request by nirmoy**
Fixes: #2880

Signed-off-by: Nirmoy Das <ndas@suse.de>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4230)

### CODE REVIEW

**Review by joestringer — APPROVED** [review]
*Updated, I noticed something on a second pass. See below*

File: cilium/cmd/status.go [hunk 1]
```diff
@@ -69,7 +69,11 @@ func statusDaemon() {
 		healthLines = 0
 	}
 	if resp, err := client.Daemon.GetHealthz(nil); err != nil {
-		fmt.Fprintf(os.Stderr, "%s\n", pkg.Hint(err))
+		if brief {
+			fmt.Fprintf(os.Stderr, "%s\n", "cilium-health: daemon unreachable")
```
Comment (joestringer) [inline cilium/cmd/status.go]: This is potentially misleading. `cilium-health` is a different daemon. This is trying to get the health from the `cilium-agent` daemon, it should say something more like `cilium: daemon unreachable`.

**Comment by manalibhutiyani** [comment]
test-me-please

**Review by joestringer — CHANGES_REQUESTED** [review]
Please update the description in the message per my above feedback, thanks!

**Comment by manalibhutiyani** [comment]
Ginkgo CI tests need to pass before merging. Initiated the tests. Thanks

File: cilium/cmd/status.go (hunk 1 above)
Comment (nirmoy) [reply cilium/cmd/status.go]: Thanks for reviewing, updated.

**Review by joestringer — APPROVED** [review]
LGTM, thanks!

**Comment by joestringer** [comment]
test-me-please

**Comment by nirmoy** [comment]
Can't understand why Ci fails :/
/home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:373
Expected
    <string>: "...      STATU..."
to equal               |
    <string>: "...     STATUS..."
/home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/vendor/github.com/onsi/gomega/internal/assertion/assertion.go:35

**Comment by joestringer** [comment]
@nirmoy looks like potentially another case of https://github.com/cilium/cilium/issues/4255. I'm working on a fix.

**Comment by joestringer** [comment]
test-me-please

**Comment by nirmoy** [comment]
test-me-please, rebased

**Comment by tgraf** [comment]
test-me-please

**Review by tgraf — APPROVED** [review]

**Pull request merged**